bot.js

```diff
@@ -206,11 +206,13 @@ function onClaimEvent (data) {
 Proof Url: ${data.optionUrl ? ('<' + data.optionUrl + '>') : 'not found'}
 Tx: <${etherscanlink}>
 ${data.gitHubIssue ? ("Task: " + displayIssue(data.gitHubIssue)) : 'not found'}
+Medal: https://provable.dev/medals/?volunteer=${data.claimreceiver}&taskid=${data.taskid}&blockNumber=${data.blockNumber}
```

Comment:
See
https://github.com/provable-dev/monitoring-bot/blob/main/bot.js#L179
On `onVoteEvent`, if `winnerIndex || winnerIndex === 0`, show the medal link in both `msg_discord` and `msg_twitter`. Otherwise, don't.

So instead of doing this on `onClaimEvent`, do it `onVoteEvent`, because simply claiming a task does not mean you won it.